src/сomponents/Person/Person.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  partner,
+}) => (
+  <div className={Person}>
+    {name && <h2>{`My name is ${name}`}</h2>}
+    {age && <p>{`I am ${age}`}</p>}
+    <p>
+      { partner
+        ? `My wife's name is ${partner}`
+        : `I am not married`
+      }
+    </p>
+  </div>
+);
+
+Person.propTypes = {
+  name: PropTypes.string.isRequired,
+  age: PropTypes.number,
+  partner: PropTypes.string.isRequired,
+};
+
+Person.defaultProps = {
+  age: false,
```

Comment:
```suggestion
  age: null,
```
age - це в нас number. Не варто давати йому булеве значення по дефолту

Reply:
зрозумів